### ISSUE TITLE
FF35 'offset is undefined'

### ISSUE THREAD

**Issue by Lucianovici** [bfirsh/needle#39]
Hi, 

I tried to investigate but without any luck. It seems strange that the error appears only if I'm using FF35, however on FF34 everything works well.

I think the problem might be here: https://github.com/bfirsh/needle/blob/master/needle/driver.py#L47

Can this be a selenium issue?
Any help is appreciated. Thanks.

```
File "/usr/lib/python2.7/unittest/case.py", line 331, in run
    testMethod()
  File "/var/lib/jenkins/jobs/pull-request-functional-tests-builder/workspace/tests/ft/b2c/tests_headers.py", line 19, in test_navbar_wrapper
    threshold=0
  File "/var/lib/jenkins/jobs/pull-request-functional-tests-builder/workspace/tests/ft/base_selenium.py", line 73, in assert_responsive
    kwargs.get("threshold", 0)
  File "/var/lib/jenkins/shiningpanda/jobs/3312707c/virtualenvs/d41d8cd9/local/lib/python2.7/site-packages/needle/cases.py", line 128, in assertScreenshot
    pass
  File "/usr/lib/python2.7/contextlib.py", line 24, in __exit__
    self.gen.next()
  File "/var/lib/jenkins/shiningpanda/jobs/3312707c/virtualenvs/d41d8cd9/local/lib/python2.7/site-packages/needle/cases.py", line 196, in compareScreenshot
    element.get_screenshot().save(output_file)
  File "/var/lib/jenkins/shiningpanda/jobs/3312707c/virtualenvs/d41d8cd9/local/lib/python2.7/site-packages/needle/driver.py", line 61, in get_screenshot
    d = self.get_dimensions()
  File "/var/lib/jenkins/shiningpanda/jobs/3312707c/virtualenvs/d41d8cd9/local/lib/python2.7/site-packages/needle/driver.py", line 55, in get_dimensions
    """, self)
  File "/var/lib/jenkins/shiningpanda/jobs/3312707c/virtualenvs/d41d8cd9/local/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 401, in execute_script
    {'script': script, 'args':converted_args})['value']
  File "/var/lib/jenkins/shiningpanda/jobs/3312707c/virtualenvs/d41d8cd9/local/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 173, in execute
    self.error_handler.check_response(response)
  File "/var/lib/jenkins/shiningpanda/jobs/3312707c/virtualenvs/d41d8cd9/local/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 166, in check_response
    raise exception_class(message, screen, stacktrace)
'Message: u\'offset is undefined\' ;
```

**Comment by jphalip** [comment]
This part of the code loads jQuery, which then runs the "offset()" method. So if you get this error it means that the object returned by `$(arguments[0])` doesn't have an `offset` method.

This is odd. Could you try manually running some tests in your browser to see if the issue is in fact with an incompatibility between jQuery and FF34?

Also, this reminds me that we should probably stop using jQuery for this altogether, and instead use the native implementation as suggested in #29. Could you try using that pull request, see if that also resolves your issue?

**Comment by Lucianovici** [comment]
Thank you @jphalip 
That PR is working perfectly with FF35. I just monkey patched `NeedleWebElement.get_dimensions` for now.
Please let me know if you decide to merge it.

**Comment by Lucianovici** [comment]
I think I also tested backwards compatibility :) Since I took the screens with jquery and tested against the one that is taking the element by native element location.

**Comment by jphalip** [comment]
Great, thank you for checking. This is now fixed in v0.2.4!
